lib/renderer/web-view/web-view.js

```diff
@@ -379,7 +379,8 @@ var registerWebViewElement = function () {
     'downloadURL',
     'inspectServiceWorker',
     'print',
-    'printToPDF'
+    'printToPDF',
+    'showDefinitionForSelection'
```

Comment:
Adding new APIs to `<webview>` allows us to do some optimizations, `<webview>.getWebContents()` is too heavy.